recipes/demjson/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "demjson" %}
+{% set version = "2.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 31de2038a0fdd9c4c11f8bf3b13fe77bc2a128307f965c8d5fb4dc6d6f6beb79
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - demjson
+
+about:
+  home: http://deron.meranda.us/python/demjson/
+  license: LGPL-3.0-or-later
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: 'Provides methods for encoding and decoding JSON formatted data'
```

Comment:
maybe add `dev_url: https://github.com/dmeranda/demjson`. Also note that the `home` url seems to be dead, even though that seems to still be the official one.

Reply:
Yeah, I realized that about the home URL.  It is probably what they want though